packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -212,6 +213,33 @@ describe("get attribute(s)", () => {
             expect(attributeIds).toContain(relationshipAttributeId);
             expect(attributeIds).toStrictEqual(expect.arrayContaining(identityAttributeIds));
         });
+
+        test("should allow to get only defaul attributes", async function () {
+            const result = await services1.consumption.attributes.getAttributes({
+                query: { default: "true" }
+            });
+            expect(result).toBeSuccessful();
+
+            const attributes = result.value;
+            expect(attributes).toHaveLength(2);
+
+            const attributeIds = attributes.map((attr) => attr.id);
+            expect(attributeIds).toContain(identityAttributeIds[0]);
+            expect(attributeIds).toContain(identityAttributeIds[1]);
+            expect(attributeIds).not.toContain(relationshipAttributeId);
+        });
+
+        test("should allow not to get defaul attributes", async function () {
+            const result = await services1.consumption.attributes.getAttributes({
+                query: { default: "!true" }
```

Comment:
Shouldn't this be `{ default: { not: true } }`? According to https://github.com/js-soft/ts-documentdb-access/tree/main/packages/querytranslator at least.

Reply:
![image](https://github.com/user-attachments/assets/17306ba3-4781-4415-9776-6a038a93c901)
@Milena-Czierlinski your query is totally fine.